storybook/src/docs/space.docs.mdx

```diff
@@ -16,12 +16,12 @@ Or at the 4 edges of a component: at the top, bottom, start, and end of its cont
 as used for buttons and input fields, and for larger components with a coloured background or border.
 
 This type of space is based on the font size – the medium space length equals the default Paragraph font size.
-Because our typography is fluid, this type of spacing is as well.
 We offer 5 lengths.
 
 ### Spacious
 
-In Spacious Mode, the medium space grows from 18 to 24 pixels between window widths of 320 and 1600.
+Because our typography is fluid in Spacious Mode, spacing is as well.
```

Comment:
Except for compact mode?

Reply:
Yes, this is in the ‘Spacious’ section. And:

> our typography is fluid **in Spacious Mode**